src/components/transaction/DataFormat.vue

```diff
@@ -29,7 +29,7 @@ export default defineComponent({
         const transferData = computed(() => actionData.value as TransferData);
         const clientHeight = computed(() => dataBox.value?.clientHeight ?? 0);
         let currentData = ref<string | unknown>(null);
-        const maxHeight = 57; // the maximum row height
+        const maxHeight = 77; // the maximum row height
```

Comment:
```suggestion
        const maxHeight = 65; // the maximum row height
```

I would suggest changing the maxHeight to 65 since it if is too big it will cause this not desired effect
![image](https://github.com/telosnetwork/open-block-explorer/assets/4420760/2ac296df-dd2e-438b-8b3a-01a2856f3d6b)